src/VirtoCommerce.ElasticAppSearch.Core/Models/Api/Search/Query/SearchQuery.cs

```diff
@@ -17,11 +18,15 @@ public record SearchQuery
 
     [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
     public IFilters Filters { get; init; }
-    
+
     [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
     [CustomJsonProperty(EmptyValueHandling = EmptyValueHandling.Ignore)]
     public Dictionary<string, SearchFieldValue> SearchFields { get; init; }
 
     [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
     public Page Page { get; init; }
+
+    [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
+    [CustomJsonProperty(EmptyValueHandling = EmptyValueHandling.Ignore)]
+    public Dictionary<string, Facet> Facets { get; set; } = new();

```

Comment:
Use `ini` and you probably don't need to initialize any properties for records of API because all of them will be overwritten due to Search -> API models convertation